src/4.6/en/textui.xml

```diff
@@ -159,7 +159,7 @@ Test Execution Options:
   --no-globals-backup       Do not backup and restore $GLOBALS for each test.
   --static-backup           Backup and restore static attributes for each test.
 
-  --colors=<flag>           Use colors in output ("never", "auto" or "always").
+  --colors=<flag>           Use colours in output ("never", "auto" or "always").
```

Comment:
This is the output from `phpunit --help`, unless you send a pull request to [PHPUnit](https://github.com/sebastianbergmann/phpunit/) it cannot be changed.